src/test/java/org/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -856,7 +856,7 @@ public void testReadMultiple() throws Exception {
      */
     @Test
     public void testWriteMultiple() throws IOException, TransformerException {
-        List<BibEntry> l = new LinkedList<>();
+        List<BibEntry> l = new ArrayList<>();
```

Comment:
I know, is not your code, but when you are here, you could provide at least a better name for that list.